querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/support/RenameMapping.java

```diff
@@ -23,9 +23,9 @@
  */
 public class RenameMapping implements Mapping {
 
-    String fromSchema, fromTable, fromColumn;
+    public String fromSchema, fromTable, fromColumn;
```

Comment:
Also the setters of the property you configure.
http://codefeed.com/tutorial/ant_config.html